packages/components/jest.setup.ts

```diff
@@ -1 +1,13 @@
 import "@testing-library/jest-dom";
+
+// Ignore console.error calls in jest tests
+beforeEach(() => {
+  jest.spyOn(console, "error").mockImplementation(() => null);
+});
+
+// Restore all sandbox spies/mocks and clean up fixture DOM after _each_ test
+afterEach(() => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore jest.spyOn adds this functionallity
+  console.error.mockRestore();
+});
```

Comment:
I forget why we need these. Can remove them?

Reply:
we test throwing an error in `Checkbox.spec.tsx`, so without this, Jest surfaces the console error in `yarn test` and it's kinda messy
![Screen Shot 2022-01-12 at 6 32 58 PM](https://user-images.githubusercontent.com/15840841/149239706-faffe502-6ae5-4711-8ebe-11a37c9995b8.png)

I should cleanup the comments some though

Reply:
hmm ok here's the test for reference:
```
  test("throws an error if no label or aria-label", () => {
    expect(() => {
      render(<Checkbox />);
    }).toThrow(/must provide a visible label or aria-label/);
  });
```

The test passes regardless of mocking `console.error` or not, so I was referencing https://github.com/facebook/jest/pull/5267#issuecomment-356605468.

I'm down to make all console error calls throw for consistency w/ the other repos though! I figured we might test for thrown errors in more components so it made sense to handle it globally

Reply:
Oh, okay. I didn't fully understand what was happening here, @anniehu4.

This is testing that an error was actually thrown, and for some reason that still gets `console.error`-ed. And to keep non-actionable stuff like that from being printed, we suppress the console.error.

I'd personally mock as necessary, and if it starts being every damn component we could move to global. But if being global could make sense, as well.